### ISSUE TITLE
Document editing widget

### ISSUE THREAD

**Issue by gsantner** [gsantner/markor#156]
#### General information

* **App version:** 0.3.0
* **System:** 7.0.1

#### Description

Add an second widget to Markor. This should be a notepad widget, which should allow highlighted syntax editing. 

TextModuleActions-Bar should not get added. The user has to choose a document which should be displayed, and the document-highlighting-type (plain,md,todo,..).

Note that the document may get changed by Markor or any other app, so be sure to keep it updated.

**Comment by gsantner** [comment]
has somebody started working on this? if so - feel free to send an PR and I will happy review it.

**Comment by gsantner** [comment]
Not planned as widget anymore.

Reason is that files need constantly need to be checked if changed and reloaded. This costs much ressources and I don't see it fitting for "lightweight app" :smile: .

**Comment by 7RST1** [comment]
Very sad this was discontinued. I have a second notes-app just for showing text on my home screen. It would be awesome if this was integrated. It is literally the only thing this app is missing.

**Comment by gsantner** [comment]
feel free (anybody who reads this) to create it.

**Comment by gsantner** [comment]
Closing in favor of #1997 which is a duplicate per se, but bit more descriptive

**Comment by jfab20** [comment]
Hi! Is this closed because this is already implemented? I don't see this feature anywhere in my app. Is this feature planned for a future update? It would be really nice!

**Comment by gsantner** [comment]
no, there was some pending work but it was not continued. Open for PRs!
